daisy/prow_images/functional_tests/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:stretch
+
+ENV GOOGLE_APPLICATION_CREDENTIALS /etc/service-account/service-account.json
+ENV GCLOUD_VERSION 168.0.0
+ENV PATH "/google-cloud-sdk/bin:${PATH}"
+
+RUN apt-get update
+RUN apt-get upgrade -y
+RUN apt-get install -y \
+    git \
+    golang \
+    python \
+    python-pip \
+    wget
+
+RUN pip install google-auth google-cloud-storage
+
+RUN wget -q https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-$GCLOUD_VERSION-linux-x86_64.tar.gz && \
```

Comment:
Why do you download gcloud? you don't use it at all as you use the google-auth google-cloud-storage libs